kevm

```diff
@@ -52,8 +53,8 @@ run_kevm_pyk() {
     local pyk_args
     pyk_args=(--definition "${backend_dir}")
     pyk_args+=(-I "${INSTALL_INCLUDE}/kframework" -I "${plugin_include}/kframework")
-    ! ${debug} || pyk_args+=(--debug)
-    execute python3 -m kevm_pyk "${pyk_args[@]}" "$@"
```

Comment:
Did we drop `debug` intentionally?

Reply:
Yes. Use `--verbose` for the timing information. `--debug` isn't used by the python tools anymore.

Reply:
Aah. I had missed the `set -x` line and thought `debug` was unused.